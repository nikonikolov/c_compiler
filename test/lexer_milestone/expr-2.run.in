
int f()
{
int x=0 && 0;
int u=5 || x;
 return u;
}
