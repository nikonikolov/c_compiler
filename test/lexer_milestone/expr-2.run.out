int Keyword Tint 2  2
f Identifier TIdentifier 2  2
( Operator T( 2  2
) Operator T) 2  2
{ Operator T{ 3  3
int Keyword Tint 4  4
x Identifier TIdentifier 4  4
= Operator T= 4  4
0 Constant TIntConstant 4  4
&& Operator T&& 4  4
0 Constant TIntConstant 4  4
; Operator T; 4  4
int Keyword Tint 5  5
u Identifier TIdentifier 5  5
= Operator T= 5  5
5 Constant TIntConstant 5  5
|| Operator T|| 5  5
x Identifier TIdentifier 5  5
; Operator T; 5  5
return Keyword Treturn 6  6
u Identifier TIdentifier 6  6
; Operator T; 6  6
} Operator T} 7  7
